.github/workflows/build-and-test-pr.yaml

```
@@ -36,12 +36,21 @@ jobs:
           target-packages: ${{ steps.get-modified-packages.outputs.modified-packages }}
           build-depends-repos: build_depends.repos
 
-      - name: Upload coverage to Codecov
-        if: ${{ steps.get-modified-packages.outputs.modified-packages != '' }}
+      - name: Check the coverage file existence
+        id: check-file-existence
+        uses: autowarefoundation/autoware-github-actions/check-file-existence@tier4/proposal
+        with:
+          file-path: |
+            lcov/total_coverage.info
+            coveragepy/.coverage
+          condition: or
+
+      - name: Upload coverage to CodeCov
+        if: steps.check-file-existence.outputs.exists == "true"
```

To make the style consistent, and I remember we can use only `''` in this.

```suggestion
        if: ${{ steps.check-file-existence.outputs.exists == 'true' }}
```